go-controller/pkg/node/gateway_localnet.go

```diff
@@ -235,7 +235,7 @@ func localnetIptRules(svc *kapi.Service) []iptRule {
 	rules := make([]iptRule, 0)
 	for _, svcPort := range svc.Spec.Ports {
 		protocol := svcPort.Protocol
-		if protocol != kapi.ProtocolUDP && protocol != kapi.ProtocolTCP {
+		if protocol != kapi.ProtocolUDP && protocol != kapi.ProtocolTCP && protocol != kapi.ProtocolSCTP {
 			protocol = kapi.ProtocolTCP
```

Comment:
This shouldn't fall back. If we don't recognize the protocol, that's an error.

OTOH you could also just remove all these checks entirely, since we now support UDP, TCP, and SCTP, and there is not likely to ever be any further protocol value.

If you keep it there should probably be a `util.KnownProtocol()` function or something, used everywhere

Reply:
Yeah I wanted to ask about this. This was already in the code and I wasn't sure why. If you and others are OK with it I would like to remove it as well.